sw/device/tests/crypto/cryptotest/firmware/BUILD

```diff
@@ -275,43 +274,53 @@ cc_library(
     ],
 )
 
-opentitan_binary(
-    name = "firmware",
-    testonly = True,
-    srcs = [":firmware.c"],
-    exec_env = [
-        "//hw/top_earlgrey:fpga_cw310_test_rom",
-    ],
-    deps = [
-        ":aes",
-        ":aes_sca",
-        ":drbg",
-        ":ecdh",
-        ":ecdsa",
-        ":hash",
-        ":hmac",
-        ":ibex_fi",
-        ":kmac_sca",
-        ":otbn_fi",
-        ":prng_sca",
-        ":sha3_sca",
-        ":trigger_sca",
-        "//sw/device/lib/base:csr",
-        "//sw/device/lib/base:status",
-        "//sw/device/lib/crypto/drivers:entropy",
-        "//sw/device/lib/testing/test_framework:check",
-        "//sw/device/lib/testing/test_framework:ottf_main",
-        "//sw/device/lib/testing/test_framework:ujson_ottf",
-        "//sw/device/lib/ujson",
-        "//sw/device/tests/crypto/cryptotest/json:aes_commands",
-        "//sw/device/tests/crypto/cryptotest/json:commands",
-        "//sw/device/tests/crypto/cryptotest/json:drbg_commands",
-        "//sw/device/tests/crypto/cryptotest/json:ecdh_commands",
-        "//sw/device/tests/crypto/cryptotest/json:hash_commands",
-        "//sw/device/tests/crypto/cryptotest/json:hmac_commands",
-        "//sw/device/tests/crypto/cryptotest/json:ibex_fi_commands",
-    ],
-)
+FIRMWARE_DEPS = [
+    ":aes",
+    ":aes_sca",
+    ":drbg",
+    ":ecdh",
+    ":ecdsa",
+    ":hash",
+    ":hmac",
+    ":ibex_fi",
+    ":kmac_sca",
+    ":otbn_fi",
+    ":prng_sca",
+    ":sha3_sca",
+    ":trigger_sca",
+    "//sw/device/lib/base:csr",
+    "//sw/device/lib/base:status",
+    "//sw/device/lib/crypto/drivers:entropy",
+    "//sw/device/lib/testing/test_framework:check",
+    "//sw/device/lib/testing/test_framework:ottf_main",
+    "//sw/device/lib/testing/test_framework:ujson_ottf",
+    "//sw/device/lib/ujson",
+    "//sw/device/tests/crypto/cryptotest/json:aes_commands",
+    "//sw/device/tests/crypto/cryptotest/json:commands",
+    "//sw/device/tests/crypto/cryptotest/json:drbg_commands",
+    "//sw/device/tests/crypto/cryptotest/json:ecdh_commands",
+    "//sw/device/tests/crypto/cryptotest/json:hash_commands",
+    "//sw/device/tests/crypto/cryptotest/json:hmac_commands",
+    "//sw/device/tests/crypto/cryptotest/json:ibex_fi_commands",
```

Comment:
```suggestion
    "//sw/device/tests/crypto/cryptotest/json:commands",
```

Reply:
Assuming it is `"//sw/device/tests/crypto/cryptotest/json/commands"` what you meant?

Reply:
Done.